stack-shell.nix

```diff
@@ -1,21 +1,4 @@
 { system ? builtins.currentSystem
 , config ? {}
 }:
-
-let
-
-  nixpkgs = builtins.fetchTarball {
-    url    = "https://github.com/NixOS/nixpkgs/archive/6054276a8e6e877f8846c79b0779e3310c495a6b.tar.gz";
-    sha256 = "11mv8an4zikh2hybn11znqcbxazqq32byvvvaymy2xkpww2jnkxp";
-  };
-
-  pkgs = import nixpkgs { inherit config; };
-
-in
-  with pkgs;
-
-  haskell.lib.buildStackProject {
-    name = "cardano-shell-env";
-    buildInputs = [ zlib openssl git ];
-    ghc = haskell.packages.ghc863.ghc;
-  }
+(import ./default.nix { inherit system config; }).stack-env
```

Comment:
* Why not keep the `haskell.lib.buildStackProject` in here?
* Import `default.nix` and use it to get the pinned `pkgs`.
* Remove the `system` and `config` args from this file because they aren't relevant for nix-shell (this was originally my mistake).
* You should be able to get the correct version of `ghc` from the haskell.nix package set.

Reply:
so with `buildStackProject` I got this error when running `satck build`:
```
/home/jbgi/.stack/setup-exe-cache/x86_64-linux-nix/Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.3: startProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
```
any idea?